website/frontend/templates/docs/build-windows.html

```diff
@@ -142,6 +144,11 @@ <h3 id="python2_packages">Python 2 packages</h3>
       <pre>pip2 install Mutagen>=1.22
 pip2 install discid</pre>
 
+      <p>
+        <b>Note:</b> If you get an error message "'pip2' is not recognized as an internal or external command, operable program or batch file." try fixing this with:
```

Comment:
I don't really like keeping workarounds for issues with components outside of our control documented here - is installing Pip something that's optional during the Python setup and can we just document to tick that checkbox?

Reply:
AFAICR I ticked the Install Pip checkbox when installing Py2, but I still needed to run this despite doing so.

It is a simple note - to save readers having to research this fix if they experience the same problem (which every new install of Py2 on Windows will experience).

I have a virtual W10 image so when I get the time I will try installing Py2 on a clean build and see what happens.

In the mean time can we leave this in.

Reply:
I agree with @mineo that this is something unrelated to Picard setup. I don't think we should include this.

Reply:
These "how to" developer pages are aimed at a small subset of users and are fairly informal anyway, so IMO we could be a bit more relaxed about such things. But if you insist I will (of course) take it out.

Reply:
I'd like to clarify my outstanding "Changes requested" review on this PR: I think this sentence is not a major problem, but `Riverside` instead  of `Riverbank` Computing is :)

Reply:
Riverside -> StreamEdge